spec/components/playbacktimelabel.spec.ts

```diff
@@ -61,10 +62,14 @@ describe('PlaybackTimeLabel', () => {
       jest.spyOn(playerMock, 'getCurrentTime').mockReturnValue(30);
 
       playbackTimeLabel.configure(playerMock, uiInstanceManagerMock);
+      playerMock.eventEmitter.fireEvent<PlayerEventBase>(
+        {type: PlayerEvent.Ready, timestamp: Date.now()}
+      );
```

Comment:
Please add the event emitting of the player mock to the `PlayerEventEmitter.ts` as we do for all other events. **(All occurrences)**